constants_local.xml

```diff
@@ -103,6 +103,7 @@
         <REMOVENODEFROMBLACKLIST_DELAY_IN_SECONDS>1</REMOVENODEFROMBLACKLIST_DELAY_IN_SECONDS>
     </epoch_timing>
     <gas>
+        <MIN_ETH_GAS_PRICE_IN_WEI>21000</MIN_ETH_GAS_PRICE_IN_WEI>
```

Comment:
This suspiciously looks like min txn gas, rather than its price. Are you sure about it?